README.md

```diff
@@ -1,4 +1,27 @@
-# Gateway API Inference Extension 
+# Gateway API Inference Extension (GIE)
+
+This project offers tools for [AI Inference], enabling developers to build [AI Gateways].
+
+[AI Inference]:https://www.digitalocean.com/community/tutorials/llm-inference-optimization
+[AI Gateways]:#concepts-and-definitions
+
+## Concepts and Definitions
+
+AI/ML is changing rapidly, and [Inference] goes beyond basic networking to include complex traffic routing and optimizations. Below are key terms for developers:
+
+- **Scheduler**: Makes decisions about which endpoint is optimal (best cost / best performance) for an inference request based on `Metrics and Capabilities` from [Model Serving Platforms].
+- **Metrics and Capabilities**: Data provided by model serving platforms about performance, availability and capabilities to optimize routing. Includes things like [Prefix Cache] status or [LoRA Adapters] availability.
+- **AI Gateway**: A proxy/load-balancer which has been coupled with a `Scheduler` to ensure optimized routing decisions for inference requests.
```

Comment:
Per the discussion in the technical overview, this is the definition of an "Inference Gateway", not an "AI Gateway."